containers/images/redis-sentinel/README.md

```diff
@@ -0,0 +1,66 @@
+# Redis sentinel packaged by SourceMation
+
+Redis is a performant in-memory key-value database. Its simplicity, accompanied
```

Comment:
Write 2-3 sentence about Sentinel itself. That's it's redis HA solution that is responsible for quorum etc.